src/db-interface.cpp

```diff
@@ -12,13 +12,24 @@ void DBInterface::connect(std::string host, std::string password) {
         try {
             static pqxx::connection c(buffer);
             con = &c;
+            con->prepare(
+                    "get_uuid_from_instanceid",
+                    "SELECT a.uuid "
+                    "FROM attachedfiles a, resources r "
+                    "WHERE r.internalid = a.id AND r.publicid = $1;"
+                    );
+            con->prepare(
+                    "update_checksum",
+                    "UPDATE attachedfiles "
+                    "SET uncompressedsize = $1, compressedsize = $2, uncompressedhash = $3, compressedhash = $4 "
+                    "WHERE uuid = $5;"
+                    );
```

Comment:
This does what?

Reply:
Set the template for prepared statements. Probably best to put it in its own function and call when initializing the connection to database.

Reply:
I was just thinking it could use some comments. If it should be a separate function then obviously do that, but I think if it only ever happens upon connection it'd be fine to leave.